index.js

```diff
@@ -8,6 +8,7 @@ var Omise = require('42-cent-omise').factory;
 var Stripe = require('42-cent-stripe').factory;
 var Braintree = require('42-cent-braintree').factory;
 var PayPal = require('42-cent-paypal').factory;
+var Payeezy = require('42-cent-payeezy').factory
```

Comment:
The module `42-cent-payeezy` doesn't exist.  
The big of the work in adding Payeezy support is to create this module.